movieapp/src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,15 +1,14 @@
 type MovieProps = {
-  key: string;
   title: string;
   description: string;
 };
 
-export function MovieCard({ key, title = 'Default movie title', description = 'Description will be added soon' }: MovieProps): JSX.Element {
+export function MovieCard(props: MovieProps): JSX.Element {
```

Comment:
Why is it MovieCard?

it renderes div with title and description. So it acn be just Card for any other content (not just Movie) 

think about components as entities that can be used in any other project (and that's why naming should be super common)

Reply:
fixed